Cargo.toml

```diff
@@ -98,6 +98,9 @@ fuel-types = "0.62"
 fuel-tx = "0.62"
 fuel-vm = "0.62"
 
+# Dependencies for Fuel Telemetry
```

Comment:
We should probably publish to crates io and point to version there.
If not, we should do a release and specify the `tag` instead of `branch` since its not apparent (at glance) which release/commit the dependency is pinned to (without looking at cargo.lock).

Reply:
Yes of course, i am doing that it is in the PR description